src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form class="search-bar search-bar_big" data-qa="big">
```

Comment:
```suggestion
    <form class="search-bar
```
![Screenshot from 2023-03-17 12-49-17](https://user-images.githubusercontent.com/33029864/226115613-d5c18fec-edfa-4851-b8f0-3f650b438069.png)
![Screenshot from 2023-03-17 12-49-02](https://user-images.githubusercontent.com/33029864/226115617-ec87833c-13b8-4dea-9a05-bc7519cc2c97.png)
" data-qa="big">

Reply:
check all cases

Reply:
Thank you,
Done :)

Reply:
Thanks a lot

Reply:
Could you please check the amended code.
Thanks in advance